bin/my_dodrio_quota.py

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env python
+# -*- coding: latin-1 -*-
```

Comment:
why is this line needed?

Reply:
don't know, as you could have guessed, i didn't write the copyright statement myself but I copied it from another script, which happened to have that line :smile:

Reply:
remove it, luis his name is not in here ;)